gcloud/bigquery/table.py

```diff
@@ -419,6 +419,8 @@ def _build_resource(self):
         if self.view_query is not None:
             view = resource['view'] = {}
             view['query'] = self.view_query
+            # https://github.com/GoogleCloudPlatform/gcloud-python/issues/1701
```

Comment:
Something mimicking the error message "Schema field shouldn't be used as input with a view" would be better